rules_points.md

```diff
@@ -29,3 +29,11 @@ For existing monster, convert Challenge Rating (CR) to a point cost based on a c
 | CR = Party Level - 5 | 0.5 |
 
 I'd worry more about that ugly 1.33 value if I knew these points would change as I test this system and get feedback from DMs and designers.
+
+## Example
+Suppose you have a party of 4 adventurers at level 5 each. If you wanted to plan a Tough encounter, 
+you would have a budget of 2 points per character, for a total of 8 points. This means you could have them 
```

Comment:
Where did you take the `2 points per character` from? I understand how to calculate points for monsters but not for characters.